src/App.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Header from './components/Header';
+import Editor from './components/Editor';
+import Canvas from './components/Canvas';
+import './styles/App.scss';
+import {TInfographicsSection} from './types/TInterfaceSection';
+
+interface IState {
+    infographicsSections: TInfographicsSection[],
+    numberOfAddedSections: number,
+    selectedSectionId: number
+}
+
+class App extends React.Component<{}, IState> {
+
+    state = {
+        infographicsSections: [] as TInfographicsSection[],
+        numberOfAddedSections: 0,
+        selectedSectionId: -1
+    };
+
+    public render() {
+
+        const {infographicsSections, selectedSectionId} = this.state;
+
+        return (
+            <div className="app">
+                <Header/>
+                <div className="app-wrapper">
+                    <Editor
+                        infographicsSections={infographicsSections}
+                        selectedSectionId={selectedSectionId}
+                        addSection={this.handleAddSection}
+                        deleteSection={this.handleDeleteSection}
+                    >
+                    </Editor>
+                    <Canvas
+                        infographicsSections={infographicsSections}
+                        selectedSectionId={selectedSectionId}
+                        handleSelectSection={this.handleSelectSection}
+                    />
+                </div>
+            </div>
+        );
+    }
+
+    private handleAddSection = () => {
+
+        const {infographicsSections, numberOfAddedSections, selectedSectionId} = this.state;
+
+        const newSection: TInfographicsSection = {
+            id: numberOfAddedSections,
```

Comment:
nestaci na toto `infographicsSections.length + 1`?

Reply:
@martin-cernin myslim, ze nie. najskor som to tak mala, ale potom som si uvedomila napr. takyto scenar: pridas sekciu s id 1, pridas sekciu s id 2, pridas sekciu s id 3, vymazes sekciu s id 2, velkost pola je 2 a tym padom ak pridas dalsiu sekciu tak bude mat id 3 a to sa nemoze stat lebo taka uz je.

Reply:
@MichaelaBajanova ok